docs/pages/about.html

```diff
@@ -57,15 +57,34 @@ <h2>HTML API</h2>
 		<a href="#tl-effect"><code>tl-effect</code></a>
 	</dt>
 	<dd><p>Attribute for building attribute driven side-effects in web-components.</p></dd>
+	<dt>
+		<a href="#tl-rebroadcast"><code>tl-rebroadcast</code></a>
+	</dt>
+	<dd><p>Attribute for emitting events when another event is triggered on an element.</p></dd>
+</dl>
+<p>
+	Additionally, Tram-Lite also has a set of custom elements that can be used in web components to enhance their
```

Comment:
Re-organizing to separate the attributes from the custom elements (now that we have several of them)